packages/iov-bns/src/decode.ts

```
@@ -107,6 +107,13 @@ function decodeVersionedId(versionedId: codecImpl.orm.IVersionedIDRef): Versione
   };
 }
 
+function decodeVersionedIdArray(versionedId: Uint8Array): VersionedId {
+  return {
```

Can you throw an error if the input does not have the expected length?